src/Microsoft.AspNet.Mvc.ModelBinding/BindingInfo.cs

```diff
@@ -76,11 +80,12 @@ public static BindingInfo GetBindingInfo(IEnumerable<object> attributes)
             var predicateProviders = attributes.OfType<IPropertyBindingPredicateProvider>().ToArray();
             if (predicateProviders.Length > 0)
             {
+                isBindingInfoPresent = true;
                 bindingInfo.PropertyBindingPredicateProvider = new CompositePredicateProvider(
                     predicateProviders);
             }
 
-            return bindingInfo;
+            return isBindingInfoPresent ? bindingInfo : null;
```

Comment:
This is a case where bringing back `IBinderMetadata` might have helped :trollface:. I think the way it is is fine though